### PR TITLE
Keyboard and input text examples

### PR DESCRIPTION
**Pull request by JuliusMalisauskasDB**
Keyboard and input text examples

### CODE REVIEW

File: dev/KeyboardAndInputExample.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React, { Component, View, StyleSheet, Animated } from 'react-native';
+import { Text } from '../src'
```
Comment (remanation) [inline dev/KeyboardAndInputExample.js]: use one import statement for all the components you use under '../src'
import { Text, Keyboard, InputText } from '../src';

File: src/InputText.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      cursorBlink: false
+    };
+  }
+
+  componentDidMount() {
+    var that = this;
+    this.inputInterval = setInterval(function() {
```
Comment (remanation) [inline src/InputText.js]: Can we use Animated instead?

File: src/Keyboard.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React, { Component, PropTypes, StyleSheet, Text, View } from 'react-native';
+
+import Key from './Key';
+import colors from './colors';
+
+export class Keyboard extends Component {
```
Comment (remanation) [inline src/Keyboard.js]: I think we should name it NumPad instead

File: src/Keyboard.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            }
+            </View>
+          );
+        })}
+      </View>
+    );
+  }
+}
+
+Keyboard.propTypes = {
```
Comment (remanation) [inline src/Keyboard.js]: Can be added internally to the component as static propTypes

File: src/colors.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const colors = {
```
Comment (remanation) [inline src/colors.js]: Should be put this under the Theme folder

File: dev/KeyboardAndInputExample.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import React, { Component, View, StyleSheet, Animated } from 'react-native';
```
Comment (remanation) [inline dev/KeyboardAndInputExample.js]: I think it should be divided in 2 examples. One for Numpad and one for InputText. So both is displayed in the list of components. Maybe create a NumpadInputExample that is then used on both Example screens. What do you think?

**Comment by remanation** [comment]
I think we should use the given Theme instead of using the colors directly

File: dev/InputExample.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
+import React, { Component, View, StyleSheet, Animated } from 'react-native';
```
Comment (remanation) [inline dev/InputExample.js]: Maybe the file should be named InputTextExample to reflect the Component name

File: dev/NumpadExample.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
+import React, { Component, View, StyleSheet, Animated } from 'react-native';
```
Comment (remanation) [inline dev/NumpadExample.js]: If the 2 examples are the same maybe export an Example component from one of the Example js files and use that in the other

Comment (JuliusMalisauskasDB) [inline dev/NumpadExample.js]: I added theme support, but for keyboard background I didn't find gray color so I left it as color.

**Comment by remanation** [comment]
I'm not sure if the background color should be a secoundaryBackgroundColor or if it should be a specific background color for the numpad. But lets leave it for now.

**Comment by jesperlundnielsen** [comment]
Looks great.

**Pull request merged**